### ISSUE TITLE
Accessibility issues with iOS and VoiceOver

### ISSUE THREAD

**Issue by Goopies** [moroshko/react-autosuggest#753]
This is going to be a list of a few issues, some are a bug and a feature request. 

**The Bugs**

http://codepen.io/moroshko/pen/LGNJMy

1) Use VoiceOver on iOS and navigate to the example
2) Attempt to access the autosuggestions

You won't be able to access the autosuggestions with the swipe feature, only by directly pressing the one you want. This means that you can't access these unless you're sighted and have the mobility to click on the one you want. 

I've attached a 'good' example of how you would be able to swipe to autosuggestions.

[Search iOS.zip](https://github.com/moroshko/react-autosuggest/files/4800063/Search.iOS.zip)

**The Feature**

Autosuggestions should, in some way, be announced to the screen reader.

Admittedly, iOS does not announce that an input has the 'aria-autocomplete' attribute. But the usability could be drastically improved for all users of Assistive Technology if some form of the auto complete would announce to the screen reader.

Examples are:
- Announce the first option available
- Announce how many options are available
- Announce in some way that options are available 

In my attachment from above you'll see they employ the first option, which announces the first available suggestion (which is usually the best). 

Let me know if you need more info or I can help in any way. Thanks.
